python_scripts/parameter_tuning_grid_search.py

```diff
@@ -245,7 +246,7 @@ def shorten_param(param_name):
 #   increased, one should decrease the value of `learning_rate` accordingly
 #   to preserve a good accuracy.
 #
-# The precise meaning of those two parameters will be explained in a latter
+# The precise meaning of those two parameters will be explained in a later
 # notebook.
```

Comment:
"latter" was intended (https://www.gingersoftware.com/content/grammar-rules/adjectives/later-vs-latter/) but maybe it's wrong to use "a latter" instead of "the latter".

Maybe instead we can just say:

```suggestion
# The precise meaning of those two parameters will be explained later.
```